lib/Elevate/Usage.pm

```diff
@@ -23,6 +23,7 @@ sub _OPTIONS {
       upgrade-to=s
       no-leapp
       non-interactive
+      leappbeta
```

Comment:
@cPholloway this is all I need to do?

Reply:
Should this option only be used with `--start`?  If so, can you add it to the `start_only_options` array as well?